components/GitHubEvent.tsx

```diff
@@ -31,14 +33,22 @@ interface GitHubRepo {
 }
 
 function GitHubEvent(props: GitHubEvent): JSX.Element {
-  const {type, actor, repo, payload} = props;
+
+  moment.defaultFormat = 'YYYY-MM-DD HH:mm:ss';
+
+  const {type, actor, repo, payload, created_at} = props;
+  const timePassed = moment(created_at).fromNow();
+
   const userLink = !actor.login.includes('[bot]') ? <ELink link={`https://github.com/${actor.login}`}>{`@${actor.login}`}</ELink> : actor.login;
   return (
     <>
       {/* <div className="card" style={{marginTop: "20px"}}> */}
       {/* <div className="card-body"> */}
-      {userLink} <GitHubEventAction type={type} payload={payload} />{' '}
-      <ELink link={`https://github.com/${repo.name}`}>{repo.name}</ELink>
+      <div style = {{overflow: 'hidden'}}>
```

Comment:
(An actual I'm not sure question) Why do we have hidden overflow? Could be concerning in the first place why we have it?
```suggestion
      <div>
```

Reply:
I have overflow:hidden since it will give you a little scroll bar in each box that requires more than a line to display. So using hidden hides the scroll bar, and overflow so that if it does overflow the box expand to two line